src/tests/regression/reported/CMakeLists.txt

```diff
@@ -8,3 +8,6 @@ caf_compile_executable(issue-422-send issue-422-send.F90)
 caf_compile_executable(issue-422-send-get issue-422-send-get.F90)
 caf_compile_executable(issue-488-multi-dim-cobounds issue-488-multi-dim-cobounds.f90)
 caf_compile_executable(issue-503-multidim-array-broadcast issue-503-multidim-array-broadcast.f90)
+if (gfortran_compiler AND (NOT CMAKE_Fortran_COMPILER_VERSION VERSION_LESS 7.0.0))
+	caf_compile_executable(issue-515-mimic-mpi-gatherv issue-515-mimic-mpi-gatherv.f90)
+endif()
```

Comment:
Exactly one new line at the end of the file would be nice. 😉

Reply:
I'm glad you put the guard on the Compiler version around this: It requires coarrays with allocatable components which is new in GFortran 7/OC 2.x

Reply:
Thanks for the comments, I'm trying to stick as close as possible to OC's conventions :)